core/spec/models/spree/order_spec.rb

```diff
@@ -40,6 +40,28 @@ def compute(_computable)
     end
   end
 
+  describe "#cancel!", pending: true do
+    context "with captured store credit" do
+      let!(:store_credit_payment_method) { create(:store_credit_payment_method) }
+      let(:order_total) { 500.00 }
+      let(:store_credit) { create(:store_credit, amount: order_total) }
+      let(:order) { create(:order_with_totals, user: store_credit.user, line_items_price: order_total).tap(&:update!) }
```

Comment:
Just something I grabbed from another spec. I'll find out if this is required (and should be worked back into the factory) or not (and should be removed from other specs).

https://github.com/solidusio/solidus/blob/5f4b2a818829d9ef1e616df544be53bfd6abe540/core/spec/models/spree/order_spec.rb#L1121